tests/performance/test_high_level_imports.py

```diff
@@ -0,0 +1,90 @@
+import importlib
+import time
+from contextlib import contextmanager
+from types import ModuleType
+from typing import Generator, Tuple
+
+import pytest
+
+
+@contextmanager
+def timing() -> Generator:
+    """ "Generator to quickly time operations. It can add 5ms so take that into account in elapsed time
+
+    Examples
+    --------
+
+        with timing() as t:
+            print("something")
+        elapsed = t()
+    """
+    start = time.perf_counter()
+    yield lambda: time.perf_counter() - start  # gen as lambda to calculate elapsed time
+
+
+def core_utilities() -> Tuple[ModuleType, ModuleType, ModuleType]:
+    """Return Tracing, Logging, and Metrics module"""
+    tracing = importlib.import_module("aws_lambda_powertools.tracing")
+    logging = importlib.import_module("aws_lambda_powertools.logging")
+    metrics = importlib.import_module("aws_lambda_powertools.metrics")
+
+    return tracing, logging, metrics
+
+
+@pytest.mark.perf
+def test_import_times_ceiling():
+    # GIVEN Core utilities are imported
+    # WHEN none are used
+    # THEN import and any global initialization perf should be below 30ms
+    # though we adjust to 35ms to take into account different CI machines, etc.
+    # instead of re-running tests which can lead to false positives
+    with timing() as t:
+        core_utilities()
+
+    elapsed = t()
+    if elapsed > 0.035:
+        pytest.fail(f"High level imports should be below 40ms: {elapsed}")
```

Comment:
__Nitpick__: the value here does not match the constant float the line above.

_(there's the same issue a few more times in this file)_